amicontrol/dags/ami_meter_read_dag.py

```diff
@@ -18,23 +20,54 @@ def ami_control_dag():
 
     config = AMIAdapterConfiguration.from_env()
     adapters = [
-        SentryxAdapter(config),
+        # Do not load this data into CaDC Snowflake until we get CaDC-associated credentials for the Sentryx API
```

Comment:
New DAG here. I commented out Sentryx so we don't load non-CaDC data into the CaDC Snowflake instance. I also commented out the old `transform` step, which wouldn't be used if we went with this ELT pattern.